src/AdoNetCore.AseClient/Token/DoneToken.cs

```diff
@@ -61,7 +61,11 @@ public enum DoneStatus : ushort
 
         public void Write(Stream stream, DbEnvironment env)
         {
-            throw new NotImplementedException();
+            Logger.Instance?.WriteLine($"-> {Type}: {Status} ({Count})");
+            stream.WriteByte((byte)Type);
+            stream.WriteUShort((ushort)Status);
+            stream.WriteUShort((ushort)TransactionState);
+            stream.WriteInt(Count);
         }
```

Comment:
Don't understand what this is for.

Reply:
Yeah, I think that's a random addition that can be removed

Reply:
Removed.